realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -142,6 +147,52 @@
     }
 
     /**
+     * Initializes the Realm library. It is required to call this method before interacting with any other of the Realm
+     * API's.
+     *
+     * A good place is in an {@link android.app.Application} subclass:
+     * <pre>
+     * {@code
+     * public class MyApplication extends Application {
+     *   \@Override
+     *   public void onCreate() {
+     *     super.onCreate();
+     *     Realm.init(this);
+     *   }
+     * }
+     * }
+     * </pre>
+     *
+     * Remember to register it in the {@code AndroidManifest.xml} file:
+     * <pre>
+     * {@code
+     * <?xml version="1.0" encoding="utf-8"?>
+     * <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="io.realm.example">
+     * <application android:name=".MyApplication">
+     *   // ...
+     * </application>
+     * </manifest>
+     * }
+     * </pre>
+     *
+     * @param context the Application Context.
+     * @throws IllegalArgumentException if null context is provided.
+     */
+    public static synchronized void init(Context context) {
+        if (context == null) {
+            throw new IllegalArgumentException("Non-null context required.");
+        }
+        RealmCore.loadLibrary(context);
+        BaseRealm.applicationContext = context;
```

Comment:
Or we could just call `getApplicationContext` instead. And add some javadoc for it.